charts/ingress-nginx/values.yaml

```diff
@@ -108,19 +108,20 @@ controller:
   ## This section refers to the creation of the IngressClass resource
   ## IngressClass resources are supported since k8s >= 1.18 and required since k8s >= 1.19
   ingressClassResource:
-    # -- Name of the ingressClass
+    # -- Name of the ingressClass (comma-seperated list supported)
     name: nginx
     # -- Is this ingressClass enabled or not
     enabled: true
     # -- Is this the default ingressClass for the cluster
-    default: false
+    # -- If multiple ingressClassResources are set, the default will always be the first in the list
+    default: true
```

Comment:
I don't think we should change this one.

If user want to install the default IngressClass, they can change this manually.

This avoids exceptions caused by default values ​​being overridden

Reply:
Yea, sorry that was from testing switching to make sure it honored the first one.  Will revert

Reply:
Thanks